<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music Player</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Play_pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>28</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Videos/Music-player-main/icon/play.png</normaloff>../../../../Videos/Music-player-main/icon/play.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Pause_pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>28</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Videos/Music-player-main/icon/pause.png</normaloff>../../../../Videos/Music-player-main/icon/pause.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Next_pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>28</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Videos/next.png</normaloff>../../../../Videos/next.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Previous_pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>28</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Videos/back.png</normaloff>../../../../Videos/back.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop_pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>28</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Videos/stop-button.png</normaloff>../../../../Videos/stop-button.png</iconset>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>256</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="songname_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="V_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Start_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>44</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="End_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Remove_pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>82</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeall_pushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>112</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Remove All</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Repeat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mute_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mute</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open                    Ctrl+O</string>
   </property>
  </action>
  <action name="actionPlay_Pause">
   <property name="text">
    <string>Play/Pause          Space</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop                Ctrl+Space</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionPlaylist">
   <property name="text">
    <string>Playlist</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>Previous          Alt+PgUp</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next                Alt+PgDn</string>
   </property>
  </action>
  <action name="actionUp">
   <property name="text">
    <string>Up                    Up</string>
   </property>
  </action>
  <action name="actionDown">
   <property name="text">
    <string>Down          Down</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="text">
    <string>Mute                 M</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
